scopa.py

```diff
@@ -0,0 +1,217 @@
+import random
+import tactics
+
+# all possible colors: Denari, Bastoni, Spade, Coppe
+colors = [tactics.denaro, " ", " ", " "]
+hand_size = 3
+initial_table_size = 4
+no_of_players = 2
+no_take = -99
+
+
+class Scopa:
+    # card deck
+    deck = []
+    # all hands
+    hands = []
+    # all piles
+    piles = []
+    # current table
+    table = []
+    # how many scopas
+    scopa_count = []
+    # how many points
+    points = []
+    # what was the last hand played
+    last_hand_played = -1
```

Comment:
All above variables could be self-descriptive by slightly changing their names - then comments would not be needed